client/src/components/PropertiesMap.tsx

```diff
@@ -266,11 +266,13 @@ export default class PropertiesMap extends Component<Props, State> {
                 left: "10px",
               }}
             />
-            {useNewPortfolioMethod ? (
-              <BigPortfolioWarning sizeOfPortfolio={this.state.addrsPoints.length} />
-            ) : (
-              <></>
-            )}
+            <div className="MapBanner__container">
```

Comment:
do we want this div to be always visible, even when useNewPortfolioMethod is false?

Reply:
oh yeah good point. We will eventually need the container for the filter-related banners, but for now that's not necessary so will put that inside and adjust later when needed.